[2024-12-26T22:00:01.118+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T22:00:01.147+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-26T21:00:00+00:00 [queued]>
[2024-12-26T22:00:01.160+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-26T21:00:00+00:00 [queued]>
[2024-12-26T22:00:01.161+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-26T22:00:01.185+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_produce_data> on 2024-12-26 21:00:00+00:00
[2024-12-26T22:00:01.195+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10920) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-26T22:00:01.197+0000] {standard_task_runner.py:72} INFO - Started process 10924 to run task
[2024-12-26T22:00:01.195+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'api_to_kafka_pipeline', 'fetch_and_produce_data', 'scheduled__2024-12-26T21:00:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/api_to_kafka_dag.py', '--cfg-path', '/tmp/tmp3qhd7gty']
[2024-12-26T22:00:01.199+0000] {standard_task_runner.py:105} INFO - Job 108: Subtask fetch_and_produce_data
[2024-12-26T22:00:01.276+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-26T21:00:00+00:00 [running]> on host bd7ba946b34c
[2024-12-26T22:00:01.432+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_to_kafka_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_produce_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-26T21:00:00+00:00'
[2024-12-26T22:00:01.434+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T22:00:01.473+0000] {api_to_kafka_dag.py:85} INFO - Fetching data for IBM at 5min interval
[2024-12-26T22:00:03.487+0000] {api_to_kafka_dag.py:92} ERROR - No time series data found in the response
[2024-12-26T22:00:03.488+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-26T22:00:03.504+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T22:00:03.505+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_to_kafka_pipeline, task_id=fetch_and_produce_data, run_id=scheduled__2024-12-26T21:00:00+00:00, execution_date=20241226T210000, start_date=20241226T220001, end_date=20241226T220003
[2024-12-26T22:00:03.550+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T22:00:03.596+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T22:00:03.602+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
